WORKSPACE

```diff
@@ -1,5 +1,36 @@
 workspace(name = "envoy_mobile")
 
+# TODO[fz]: Remove once envoy updates protoc
+# Uses PGV that includes the CC NOP template to disable validation.
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+http_archive(
+    name = "bazel_gazelle",
+    sha256 = "de69a09dc70417580aabf20a28619bb3ef60d038470c7cf8442fafcf627c21cb",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-gazelle/releases/download/v0.24.0/bazel-gazelle-v0.24.0.tar.gz",
+        "https://github.com/bazelbuild/bazel-gazelle/releases/download/v0.24.0/bazel-gazelle-v0.24.0.tar.gz",
+    ],
+)
+
+load("@bazel_gazelle//:deps.bzl", "go_repository")
+
+go_repository(
+    name = "com_github_lyft_protoc_gen_star",
+    importpath = "github.com/lyft/protoc-gen-star",
+    sum = "h1:xOpFu4vwmIoUeUrRuAtdCrZZymT/6AkW/bsUWA506Fo=",
+    version = "v0.6.0",
+)
+
+git_repository(
+    name = "com_envoyproxy_protoc_gen_validate",
+    commit = "79071f0f8b04188b297a0517a6e55b2d3641ab5a",
+    remote = "https://github.com/envoyproxy/protoc-gen-validate"
+)
+
+# End of PGV overrides
```

Comment:
Would this be better put in: https://github.com/envoyproxy/envoy-mobile/blob/main/bazel/envoy_mobile_repositories.bzl?

Adding it here wouldn't be able to be used if envoy mobile is a submodule (e.g like a private envoy mobile for a given org/company)